appengine/websockets/README.md

```diff
@@ -1,7 +1,7 @@
 # Node.js websockets sample for Google App Engine
 
 This sample demonstrates how to use websockets on
-[Google App Engine Managed VMs](https://cloud.google.com/appengine) with Node.js.
+[Google App Engine Flexible Environment](https://cloud.google.com/appengine) with Node.js.
 
 __Note:__ Secure WebSockets are currently not supported by Managed VMs.
```

Comment:
Should we clarify that "Managed VMs" == "App Engine Flex" here? (Or otherwise not use the term "Managed VMs"?)